ming_sim/cli_backend.py

```
@@ -397,84 +396,158 @@ def _warm_keychain() -> None:
+                    last_activity_at=last_activity, policy=policy, clock=tick,
+                )
+                continue
+            if line is None:
+                open_streams -= 1
+                continue
+            last_activity = tick()
+            if kind == "err":
+                stderr_parts.append(line)
+                continue
+            yield line
+        # 管道已 EOF 但进程未退：静默同样计入 idle 预算，仍无总墙钟。
```

Ruff (0.16.3)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)